EAD-Assignment-4/WebContent/WebContent/Scripts/script.js

```diff
@@ -0,0 +1,175 @@
+function checkValidation() {
+	if (isFNameValid() && isLNameValid() && isEmailValid() && isPasswordValid()
+			&& isConfirmPasswordValid() && isContactNumberValid() && isOrganizatioNameValid() && isDobValid()) {
+		return true;
+	} else {
+		alert("Enter valid inputs.")
+		return false;
+	}
+}
+
+function isFNameValid() {
+	var name = document.getElementById('fname').value;
+	var regexpression = new RegExp("^([a-zA-Z][a-zA-Z ]{2,})$");
+	if (regexpression.test(name)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isLNameValid() {
+	var name = document.getElementById('lname').value;
+	var regexpression = new RegExp("^([a-zA-Z]{2,})$");
+	if (regexpression.test(name)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isEmailValid() {
+	var email = document.getElementById("email").value;
+	var regexpression = new RegExp(
+			"^([a-zA-Z0-9_\.]{2,})@([a-zA-Z0-9_]{2,})\.([a-zA-Z]{2,5})$");
+	if (regexpression.test(email)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isPasswordValid() {
+	var password = document.getElementById("pwd").value;
+	var regexpression = new RegExp(
+			"^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$");
+	if (regexpression.test(password)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isConfirmPasswordValid() {
+	var password = document.getElementById("pwd").value;
+	var cpassword = document.getElementById("cpwd").value;
+	if (password == cpassword) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isDobValid() {
+	  var isValid = false;
+	  var today = new Date();
+	  var lowestAcceptableDate = new Date("01/01/1918");
+	  var passedDate = new Date(document.getElementById("dob").value);
+	  if(passedDate >= lowestAcceptableDate && passedDate <= today ) {
+      isValid = true;
+      }
+	  return isValid;
+}
+
+function isContactNumberValid() {
+	var contactno = document.getElementById("cnumber").value;
+	var regexpression = new RegExp("^([0-9]{8,})$");
+	if (regexpression.test(contactno)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+function isOrganizatioNameValid(){
+	var name= document.getElementById("orgname").value;
+	var errorname="";
+	if(name==errorname)
+	{
+		return false;
+	}
+	else{
+		return true;
+	}
+}
+
+function fnamefocusout() {
+	if (isFNameValid()) {

```

Comment:
Can Create a separate CSS class for these style and then add or remove those classes based on the conditions